scenarios/test_codegen/test/ChainManager_test.res

```diff
@@ -2,7 +2,7 @@ open Belt
 open RescriptMocha
 
 let populateChainQueuesWithRandomEvents = (~runTime=1000, ~maxBlockTime=15, ()) => {
-  let config = Config.getGenerated()
+  let config = RegisterHandlers.registerAllHandlers()
```

Comment:
Why is this change? It reruns the EventHandlers code, which I don't think current with the current testing implementation.

Reply:
Got you...

Basically the config can only be set after handlers registered. So I should change it rather to a function that if the ref is None, it runs registerAllHandlers and then returns the value else it can return the cached value.

I'm not too worried about the performance on a test though TBH.

Reply:
Did you get a case when it's missing?

Reply:
Yes, all the cases were missing.

Reply:
Hmmm, weird how it worked before

Reply:
No it's not weird, it's because I  moved the Config.setGenerated into the register handlers function. That way you can be sure you have all the wildcard validation and entries on the config.

Reply:
Ah, missed the change